vipps-checkout-api.md

```diff
@@ -305,6 +305,17 @@ Vipps Checkout will expose a polling endpoint as described in our [Swagger](http
 
 In order to determine the status of a payment, you must poll epayment as described [here](https://vippsas.github.io/vipps-epayment-api/index.html#tag/QueryPayments/operation/getPayment).
 
+## Step 3a: If a transaction is authorized, capture payment
```

Comment:
Remember to update the table of contents with the new step, and remove the old one

Reply:
Done! And fixed a broken link there as well :)